### ISSUE TITLE
[FEATURE] Right-To-Left support

### ISSUE THREAD

**Issue by abokish** [anthropics/claude-code#70034]
### Preflight Checklist

- [x] I have searched [existing requests](https://github.com/anthropics/claude-code/issues?q=is%3Aissue%20label%3Aenhancement) and this feature hasn't been requested yet
- [x] This is a single feature request (not multiple features)

### Problem Statement

when chatting with Claud in RTL language, or mixed, it is very hard to read the answer. He knows to write very good in the RTL language, but when numbers and English letters are involved in the same sentence (very often), this is almost impossible to understand his answer.

### Proposed Solution

teach him how to write from right to left numbers and English letters.
there are already private solutions for that. but i belive it should be part of Claude.
It the same for the Claude Code extention in VS code and for the Claude windows app.

### Priority

High - Significant impact on productivity

### Feature Category

Interactive mode (TUI)

### Use Case Example

1. write in any RTL language a qustion like: מה פירוש המונח RTL?
2. good luck with understanding his answer

### Additional Context

<img width="937" height="728" alt="Image" src="https://github.com/user-attachments/assets/e25bc4c4-7352-4e9a-8974-3804e6a28135" />
